com.microsoft.mrtk.accessibility/Utilities/ObjectClassification.cs

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Microsoft.MixedReality.Toolkit.Accessibility
+{
+    /// <summary>
+    /// Classifications for describable objects that may appear in the scene.
+    /// </summary>
+    [Flags]
+    public enum ObjectClassification : ushort
```

Comment:
Is there a reason this is specifically `ushort`?

Reply:
we dont need 32 bits :)

Reply:
I've never, ever heard or seen of idiomatic C# code restricting the bit width of an enum. 

<img width="566" alt="image" src="https://user-images.githubusercontent.com/5544935/190560622-f1f26fb7-734c-42e0-9bf1-d6347df8c8ab.png">

Sounds like extreme premature optimization to me... The .NET design guidelines recommend using standard integers for enums, as well, because everything is `DWORD`-aligned in-memory anyway, so there is no memory savings or implications for performance in any context other than runtime serialization or on-disk files, neither of which we're doing here.

<img width="567" alt="image" src="https://user-images.githubusercontent.com/5544935/190560803-06d6368d-d5ef-4ade-bf05-15b9e53be278.png">

Reply:
updated